doc/contributing/issues.md

```diff
@@ -60,5 +60,21 @@ activities, such as applying labels and closing/reopening/assigning issues.
 For more information on the roles and permissions, see ["Permission levels for
 repositories owned by an organization"](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization).
 
+triagers are requested to abstain from below actions:
+* approve or request changes to a pull request
+* edit wikis
+
+When triagging issues and PR:
+
+* Show patience and empathy, especially to first-time contributors.
+* Show no patience towards spam or troll, close the issue without interacting with it and
+  report the user to the moderation repository.
+* If you're not able to reproduce an issue, leave a comment asking for more info and
+  add the `needs more info` label.
```

Comment:
This label doesn't exist

Reply:
We can create it

Reply:
It can probably replace `repro exists` as an inverse.